animation_nodes/data_structures/attributes/attribute.pxd

```diff
@@ -0,0 +1,14 @@
+cdef enum AttributeType:
+    UVMAP, MATERIAL_INDEX, VERTEX_COLOR, CUSTOM
+cdef enum AttributeDomain:
+    POINT, EDGE, POLYGON, CORNER
+cdef enum AttributeDataType:
+    INT, FLOAT, FLOAT2, FLOAT_VECTOR, FLOAT_COLOR, BYTE_COLOR, BOOLEAN
+
+cdef class Attribute:
+    cdef:
+        public str name
+        public AttributeType type
+        public AttributeDomain domain
+        public AttributeDataType dataType
```

Comment:
Those should be readonly. If the domain or dataType change, the data will also need to change, so those should not be mutable.

Reply:
It is changed.